packages/plugin-lightningcss/modern.config.ts

```diff
@@ -0,0 +1,3 @@
+import baseConfig from '../../scripts/modern.base.config';
```

Comment:
usually we need to bundle the loader file, such as https://github.com/web-infra-dev/rsbuild/blob/main/packages/plugin-pug/modern.config.ts#L6-L12

Reply:
I wonder why the e2e case can pass without the `loader` bundle config.

Reply:
and it is recommended to use `buildConfigWithMjs` to generate mjs outputs.